themes/default/gulpfile.js

```diff
@@ -17,6 +17,9 @@ var gulp = require('gulp'),
 	concat = require('gulp-concat'),
 	uglify = require('gulp-uglify'),
 	plumber = require('gulp-plumber');
+	rework = require('gulp-rework'),
```

Comment:
I'm not so keen on rework being a default. It should really be an edge case thing, rather than an every project thing.  Can we just write some docs around how to use it instead?